tests/recovery/recovery_test.go

```diff
@@ -24,6 +24,9 @@ var _ = Describe("cOS Recovery upgrade tests", func() {
 
 				version := out
 				By("upgrading to raccos/releases-opensuse:cos-system-0.5.0")
+				By("turning off verification, and pointing to an old release branch")
+				out, err = s.Command("sed -i 's/verify: true/verify: false/g' /etc/luet/luet.yaml")
```

Comment:
can we make this a helper function so its clearer and reusable?

Reply:
I'd say the approach should be to get rid of all this as soon as we stop using DCT and start using other validation strategies.

Reply:
While I see the point on doing it as helper function I'd say first finalize the DCT and validation story and then adapt tests to make them cleaner. They will need some adaptations at that point too.